doc/development_guide/how_tos/custom_checkers.rst

```diff
@@ -223,6 +223,18 @@ Now we can debug our checker!
     environment variable or by adding the ``my_plugin.py``
     file to the ``pylint/checkers`` directory if running from source.
 
+    If your pylint config has an init-hook that modifies
+    ``sys.path`` to include the module's parent directory, this
+    will also work, but only if:
```

Comment:
```suggestion
    will also work, but only if either:
```

Reply:
Just saw this myself, mortified, good spot thanks @DanielNoord!